src/main/interface/cli.c

```diff
@@ -3757,7 +3757,11 @@ static void cliMsc(char *cmdline)
             mpuResetFn();
         }
 
-        *((uint32_t *)0x2001FFF0) = 0xDDDD1010; // Same location as bootloader magic but different value
+#ifdef STM32F7
+        *((__IO uint32_t*) BKPSRAM_BASE + 16) = 0xDDDD1010;
+#elif defined(STM32F4)
+        *((uint32_t *)0x2001FFF0) = 0xDDDD1010;
```

Comment:
We should really put the `0xDDDD1010` into a define so it isn't repeated all over.